### PR TITLE
'jettyStop' working

### PR DESCRIPTION
**Pull request by Gascon95**

### CODE REVIEW

**Comment by nebur395** [comment]
Please, add a brief explanation about the new command in the wiki.

File: build.gradle [hunk 1]
```diff
@@ -18,7 +18,14 @@ dependencies {
     providedCompile "javax.servlet:javax.servlet-api:3.0.1"
 }
 
-jettyRun {
+jettyRun{
```
Comment (wynro) [inline build.gradle]: Unnecessary change

**Comment by Gascon95** [comment]
README updated.

**Comment by nebur395** [comment]
LGTM 👍

File: build.gradle (hunk 1 above)
Comment (iAbadia) [inline build.gradle]: Pretty much like you comment, pal @wynro
